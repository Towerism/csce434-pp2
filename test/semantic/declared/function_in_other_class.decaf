class Asdf {
      int qwer();
}

int asdf() {
    Asdf a;
    a.qwer();
}