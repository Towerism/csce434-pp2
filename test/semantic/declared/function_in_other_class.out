
*** Error line 2.
      int qwer();
                ^
*** syntax error

